app/javascript/react/components/TrailTile.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const TrailsTile = (props) => {
+    return (
+        <div>
+            <ul>
+                <li>
+                    <Link to={`/trails/${props.id}`}> {props.name} </Link>
+                    {`${props.city} ${props.zip}`}
+                </li>
+            </ul>
```

Comment:
lots of extra HTML tags here - looks like we only need a `li` - a `ul` would be used for a collection of list items, and the div is not needed as the `li` is the top most element returned